django/contrib/postgres/functions.py

```diff
@@ -3,17 +3,9 @@
 
 class RandomUUID(Func):
     template = 'GEN_RANDOM_UUID()'
-
-    def __init__(self, output_field=None, **extra):
```

Comment:
The problem is not about `output_field`. `RandomUUID` didn't accept any source expressions previously, but it accepts any number of them now. I think that behavior should be preserved here.
Also I think it makes sense to inherit directly from `Expression` and define `as_sql()` to simply return `'GEN_RANDOM_UUID()'`.